versioned_docs/version-2.0/concepts/admin/admin.mdx

```diff
@@ -0,0 +1,45 @@
+# Admin
+
+This conceptual guide covers topics related to managing users, organizations, and workspaces within LangSmith.
+
+## Organizations
+
+An organization is a logical grouping of users within LangSmith. Typically, there is one organization per company. An organization can have multiple workspaces.
+
+## Workspaces
+
+A workspace is a logical grouping of users within an organization that share access to the same set of resources, including tracing projects, datasets, annotation queues, and prompts.
+
+## Users
+
+A user is a person who has access to LangSmith. Users can be members of one or more organizations and workspaces within those organizations.
+
+## API keys
+
+:::danger Soon to be deprecated
+API keys are being deprecated in favor of personal access tokens (PATs). We recommend using PATs for all new integrations.
+:::
+
+API keys are used to authenticate requests to the LangSmith API. They are created by users and scoped to a workspace. This means that all requests made with an API key will be associated with the workspace that the key was created in. The API key will have the ability to create, read, update, delete all resources within that workspace.
+
+API keys are prefixed with `ls__`
+
+## Personal access tokens (PATs)
+
+Personal access tokens (PATs) are used to authenticate requests to the LangSmith API. They are created by users and scoped to a user. The PAT will have the same permissions as the user that created it.
```

Comment:
and is scoped to a workspace

Reply:
We talked about how this is scoped to an org now right?